Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -23,6 +23,14 @@ to also change `ROOK_OPERATOR_NAMESPACE` to create a new Rook Operator for each
 forget to set `ROOK_CURRENT_NAMESPACE_ONLY`), or you can leave it at the same value for every
 Ceph cluster if you only wish to have one Operator manage all Ceph clusters.
 
+!!! note
```

Comment:
I'm thinking the flow will be more natural if this is just a paragraph instead of a separate note. With the note, it seems awkward that the console command is separate, but it should belong to the note. The rendering would make the console command appear separate from the note.
```suggestion
```